web/modules/custom/usagov_wizard/usagov_wizard.module

```diff
@@ -40,9 +40,12 @@ function usagov_wizard_form_taxonomy_term_wizard_form_alter(&$form, &$form_state
       ],
       '#weight' => 10,
     ];
-
+    $taxonomy_term = Term::load($form['tid']["#value"]);
+    $termParents = \Drupal::classResolver(MenuChecker::class);
+    $termParents = $termParents->getTermParents($taxonomy_term);
+    $rootTermTid = array_reverse($termParents)[0];
```

Comment:
You can avoid array_reverse (might matter if you have a lot of terms or large entities in your arrays) altogether if you use array_key_last(). Good performance discussion here: https://stackoverflow.com/questions/2348205/how-to-get-last-key-in-an-array#answer-52098132

Something like this should work:

`$rootTermTid = $term_parents[array_key_last($term_parents)]`

Reply:
Thanks, good point, I updated to reflect this change. I also realized that I don't really need to set a meta description on non-root terms, since they are noindexed anyway, so I removed some code from hook_preprocess_html()